app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -893,6 +919,29 @@ class BrowserTabFragment :
         }
     }
 
+    private fun launchAppLinkDialog(context: Context, url: String?, headers: Map<String, String>, onClick: () -> Unit) {
+        val isShowing = alertDialog?.isShowing
+
+        if (isShowing != true) {
+            alertDialog = AlertDialog.Builder(context)
+                .setTitle(R.string.appLinkDialogTitle)
+                .setMessage(getString(R.string.confirmOpenExternalApp))
+                .setPositiveButton(R.string.yes) { _, _ ->
+                    onClick()
+                    webViewClient.appLinksHandler.reset()
+                }
+                .setNegativeButton(R.string.no) { _, _ ->
+                    if (url != null) {
+                        webView?.loadUrl(url, headers)
+                        webViewClient.appLinksHandler.enterBrowserState()
+                    } else {
+                        showToast(R.string.unableToOpenLink)
```

Comment:
I like toast but only for breakfast... thought we had removed them everywhere :(

Reply:
Lol, this was copy-paste from the other dialog... I'll remove it